src/qml/views/keyframes/keyframes.qml

```diff
@@ -513,9 +513,11 @@ Rectangle {
 
         property alias text: bubbleHelpLabel.text
 
-        function show(x, y, text) {
-            bubbleHelp.x = x + tracksArea.x - tracksFlickable.contentX - bubbleHelpLabel.width;
-            bubbleHelp.y = Math.max(0, y + tracksArea.y - tracksFlickable.contentY - bubbleHelpLabel.height);
+        function show(text) {
+            var point = application.mousePos;
+            point = parent.mapFromGlobal(point.x, point.y);
+            bubbleHelp.x = point.x + 12;
+            bubbleHelp.y = point.y + 12;
```

Comment:
These 12 values are dependent on the mouse cursor size, which I doubt there is a way to get in QML, but some people will run with a larger cursor and then the bubble help appears underneath a little, which is not a big deal but worth taking into consideration instead of being tight like this.

Reply:
```
            bubbleHelp.x = point.x + 20;
            bubbleHelp.y = Math.max(point.y - 20, 0);
```
works better for me to prevent obscuring waveforms and dealing with a larger cursor size.

Reply:
I was thinking of it as a tooltip and tried to emulate that visual look/position. And in fact, I was even thinking about re-implementing the bubble help using [QToolTip](https://doc.qt.io/qt-6/qtooltip.html). But that is a bigger change, and I like the "bubble" look that we have.

Tool tips typically appear in the lower right - presumably because we read top-to-bottom/left-to-right:
![image](https://user-images.githubusercontent.com/821968/194786908-6858539a-f1e1-4341-bc54-9097bf349ea8.png)
Also, tool tips often have logic to move the box around to avoid being clipped off the screen:
![image](https://user-images.githubusercontent.com/821968/194786935-0fddb20a-9a7d-4760-a05e-8f0673d41b7e.png)
I did not bother to write extra logic to avoid clipping the bubble help box

In this specific case, I think your suggested offsets do work  better in the timeline. So I made that change. But I am open minded about it and we can tweak them. I am also willing to re-implement the  bubble help as a QToolTip - but then we will have less control over the position. However, that would increase consistency with the rest of the Shotcut UI.

Reply:
It was not intended to be a tooltip. I don’t want to reimplement many things in the GUI in the near- or mid-term.